### PR TITLE
Yt tutorials for SUS FP SF HEX FAC

### PR DESCRIPTION
**Pull request by Deppresso-Exppresso**
Just that B)

### CODE REVIEW

**Comment by kavinkul** [comment]
What with all these merge commits that I'm seeing here?

File: JSON/Forget Perspective.json [hunk 1]
```diff
@@ -57,5 +57,6 @@
   "Souvenir": { "Status": "NotACandidate" },
   "SteamID": "1831737244",
   "Symbol": "Fp",
+  "TutorialVideoUrl": "https://www.youtube.com/watch?v=X0li_huR0Eg&t",
```
Comment (kavinkul) [inline JSON/Forget Perspective.json:60]: `&t` is redundant as I believe it is for specifying the start time of the video. Perhaps remove them for consistency with others.

File: JSON/Forget Any Color.json [hunk 2]
```diff
@@ -58,5 +58,6 @@
   "Souvenir": { "Status": "Supported" },
   "SteamID": "2126185549",
   "Symbol": "Fac",
+  "TutorialVideoUrl": "https://www.youtube.com/watch?v=g6qQcHsyeBA&t",
```
Comment (kavinkul) [inline JSON/Forget Any Color.json:61]: Remove `&t`

File: JSON/Simon Forgets.json [hunk 3]
```diff
@@ -59,9 +59,6 @@
   "Souvenir": { "Status": "NotACandidate" },
   "SteamID": "1952732783",
   "Symbol": "Sf",
-  "TwitchPlays": {
-    "AutoPin": true,
-    "ScoreExplanation": "This module awards 2 point per correct input plus 3 points for every module on the bomb upon solving."
-  },
```
Comment (kavinkul) [inline JSON/Simon Forgets.json:65]: This should not be removed. This is to keep the explanation of how the module is scored as it is not simple.

File: JSON/Simon Forgets.json [hunk 4]
```diff
@@ -59,9 +59,6 @@
   "Souvenir": { "Status": "NotACandidate" },
   "SteamID": "1952732783",
   "Symbol": "Sf",
-  "TwitchPlays": {
-    "AutoPin": true,
-    "ScoreExplanation": "This module awards 2 point per correct input plus 3 points for every module on the bomb upon solving."
-  },
+  "TutorialVideoUrl": "https://www.youtube.com/watch?v=Zp3_LwOsbn0&t",
```
Comment (kavinkul) [inline JSON/Simon Forgets.json:62]: Remove `&t`

File: JSON/hexOS.json [hunk 5]
```diff
@@ -13,5 +13,6 @@
   "Souvenir": { "Status": "Supported" },
   "SteamID": "2106930521",
   "Symbol": "Hex",
+  "TutorialVideoUrl": "https://www.youtube.com/watch?v=45ap6mFhrWE&t",
```
Comment (kavinkul) [inline JSON/hexOS.json:16]: `&t`

File: JSON/Simon's Ultimate Showdown.json [hunk 6]
```diff
@@ -15,5 +15,6 @@
     "Status": "Considered"
   },
   "SteamID": "2158339043",
+  "TutorialVideoUrl": "https://youtu.be/tnBSg7k62yo",
```
Comment (kavinkul) [inline JSON/Simon's Ultimate Showdown.json:18]: No Youtube link in form of `youtu.be` Please use the link as shown on the address bar (without any extra such as `&t` of course).

**Review by kavinkul — CHANGES_REQUESTED** [review]

File: JSON/Forget Any Color.json (hunk 2 above)
Comment (Emik03) [reply JSON/Forget Any Color.json:61]: In the following days a major update will be released, changing the rules and leaving parts of this tutorial outdated. 

For now, this should be excluded.

**Comment by Deppresso-Exppresso** [comment]
I CLOSED THIS due to the &t on the yt link and the fac update and that i somehow removed a scoring system for SF

**Comment by Deppresso-Exppresso** [comment]
I will get home tomorrow by then the issues remain

**Pull request closed without merging**